### PR TITLE
Follow up PR for timestepping

### PR DESCRIPTION
**Pull request by jbathmann**
Appendinx to #136: removal of update_gd functionality as it produces bugs.

### CODE REVIEW

File: TimeLoopLib/ExternalDynamicTimeStepping.py [hunk 1]
```diff
@@ -33,21 +33,19 @@ def __init__(self, project, t_0):
     def setSteps(self, n_step_ag, n_step_bg, n_step_gd):
```
Comment (mcwimm) [inline TimeLoopLib/ExternalDynamicTimeStepping.py:33]: remove _gd in args

Comment (mcwimm) [reply TimeLoopLib/ExternalDynamicTimeStepping.py:33]: + remove in comment

Comment (jbathmann) [inline TimeLoopLib/ExternalDynamicTimeStepping.py:33]:
```suggestion
    def setSteps(self, n_step_ag, n_step_bg):
```

Comment (jbathmann) [reply TimeLoopLib/ExternalDynamicTimeStepping.py:33]: done

Comment (mcwimm) [reply TimeLoopLib/ExternalDynamicTimeStepping.py]: the same issue is in MANGA.py ln. 28+29

Comment (mcwimm) [reply TimeLoopLib/ExternalDynamicTimeStepping.py]: resolved

**Pull request merged**